CC/Sounder/receiver.cc

```diff
@@ -182,15 +182,18 @@ void Receiver::loopRecv(int tid, int core_id, SampleBuffer* rx_buffer)
 
     // read rx_offset to align the FPGA time of the BS
     // by performing dummy readStream()
-    std::vector<std::complex<int16_t>> syncbuffBs(config_->sampsPerSymbol * config_->symbolsPerFrame, 0);
+    std::vector<std::complex<int16_t>> syncbuff0Bs(config_->sampsPerSymbol * config_->symbolsPerFrame, 0);
+    std::vector<std::complex<int16_t>> syncbuff1Bs(config_->sampsPerSymbol * config_->symbolsPerFrame, 0);
     std::vector<void*> syncrxbuffBs(2);
-    syncrxbuffBs[0] = syncbuffBs.data();
+    syncrxbuffBs[0] = syncbuff0Bs.data();
+    if (bsSdrCh == 2)
+        syncrxbuffBs[1] = syncbuff1Bs.data();
 
     if (kUseUHD) {
         std::cout << "Sync BS host and FPGA timestamp..." << std::endl;
         baseRadioSet_->radioRx(0, syncrxbuffBs.data(), config_->sampsPerSymbol, rxTimeBs);
         // schedule the first beacon in the future
-        txTimeBs = rxTimeBs + config_->sampsPerSymbol * config_->symbolsPerFrame * 10;
+        txTimeBs = rxTimeBs + config_->sampsPerSymbol * config_->symbolsPerFrame * 20;
         baseRadioSet_->radioTx(0, beaconbuff.data(), 2, txTimeBs);
```

Comment:
Can we use a `flags` variable here instead of 2?

Reply:
Done